components/match_ticker/commons/match_ticker.lua

```diff
@@ -467,6 +479,21 @@ function MatchTicker.switchOpponents(match)
 	return match
 end
 
+--- Fetches region of a tournament
+---@param tournamentPage string
+---@return string?
+MatchTicker.fetchRegionOfTournament = FnUtil.memoize(function(tournamentPage)
+	local tournamentData = mw.ext.LiquipediaDB.lpdb('tournament', {
+		conditions = '[[pagename::' .. tournamentPage .. ']]',
+		query = 'region',
+		limit = 1,
+	})[1]
+	if not tournamentData or not tournamentData.locations then
+		return nil
+	end
+	return tournamentData.locations.region1
```

Comment:
would we want to add support for multi region events?
```suggestion
	local regions = Table.filterByKey(tournamentData.locations, function(key) return Staing.startsWith(key, 'region') end)
	return Array.extractValues(regions)
```

Reply:
Multi-region events would cause issues with the tournament ticker's way of filtering, so probably not